docs/connectors/connector-list.mdx

```diff
@@ -2,7 +2,7 @@
 title: "Connector List"
```

Comment:
On the connector list, we should change the release download links to go through scarf. Might be a thing for a separate PR though.

Reply:
yes that's on my list.. trying to figure out where we set the URL values so I could update them for the json list, would appreciate some quick pointers since you created that generator 👀